region/norfair/west/Ice Beam Acid Room.json

```diff
@@ -246,6 +246,43 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Getting Blue Speed, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 2,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$1.D"
+        }
+      },
+      "requires": [
+        {"or": [
+          {"and": [
+            "canBlueSpaceJump",
```

Comment:
This should have a max extra run speed.  Maybe springball needs one too.
These are reminiscent of pit room.

Reply:
Cool, just added a max speed now across both strats. The canLongChainTemporaryBlue variants might not need it, but it didn't seem worth splitting off into a separate strat